index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Javascript Society Summit - Conference Web Page</title>
+</head>
+
+<body>
+    <!--HEADER-->
+    <div class="background">
+        <header class="header">
+            <div class="dark-nav">
+                <ul>
+                    <li><a href="https://www.facebook.com/"><img src="images/facebook.png" alt="fcebook icon"></a></li>
+                    <li><a href="https://twitter.com/home"><img src="images/twitter.png" alt="twitter icon"></a></li>
+                    <li><a href="/">English</a></li>
+                    <li><a href="/">My Page</a></li>
+                    <li><a href="/">Log out</a></li>
+                </ul>
+            </div>
+
+            <nav class="navbar">
+                <a href="index.html" class="nav-branding"><img src="images/dark logo.png" alt="logo"></a>
+
+                <ul class="nav-menu">
+                    <li class="nav-item">
+                        <a href="about.html" class="nav-link orange-text">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#program" class="nav-link">Program</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#featured" class="nav-link">Featured</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#sponsor" class="nav-link">Sponsor</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="/" class="nav-link border-text">Community</a>
+                    </li>
+                </ul>
+
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </header>
+        <!--HERO-->
+        <section class="hero">
+            <p class="quote">
+                “Welcome to the javascript society summit”
+            </p>
+            <h1 class="first-heading">
+                Javascript society<br>
+                Summit<br>
+                2023
+            </h1>
+            <div class="first-para">
+                <p>
+                    An Javascript coders community that believes in the value of openness, sharing, creativity and tech
+                    empowerment. Creating a positive change with people all over the world. We are hosting Africa
+                    biggest open source conference this November, in Nairobi, Kenya.
+                </p>
+            </div>
+            <h2 class="second-heading">
+                2023.07.3(MON) ~ 7(FRI)
+            </h2>
```

Comment:
As suggested by the previous reviewer, kinly use a `p`tag for your date, as the `header`tag is only meant to be used for section headings 😄